process-migration-service/README.md

```diff
@@ -0,0 +1,150 @@
+# Process Instance Migration Service
```

Comment:
we need to add a section on how to use it. I tested base migration for evaluation process with following which could be the base for the docs. This will give users really easy way to make use of the features

1. Define plan without any node mapping

```
http://localhost:8180/plans

{
    "name": "Test plan",
    "description": "Evaluation Process Test Plan",
    "sourceContainerId": "evaluation_1.0",
    "targetProcessId": "evaluation",
    "targetContainerId": "evaluation_1.1"
}
```

2. Start Kie Server and deploy two versions of evaluation project (evaluation_1.0 and evaluation_1.1)
3. Start one instance of evaluation process from project (evaluation_1.0)
4. Create sync migration

```
http://localhost:8180/migrations

{
    "planId": 1,
    "processInstanceIds": [1],
    "kieserverId": "sample-server",
    "execution": {
   	 "type": "SYNC"
    }
}
```

5. View migration details
http://localhost:8180/migrations/1
http://localhost:8180/migrations/1/results

6. Create two additional process instances of evaluation process from project (evaluation-1.0)
7. Run migration as scheduled async operation for both instances

```
http://localhost:8180/migrations

{
    "planId": 1,
    "processInstanceIds": [3,4],
    "kieserverId": "sample-server",
    "execution": {
   	 "type": "ASYNC",
   	 "scheduledStartTime": "2018-12-11T12:35:00.000Z"
    }
}
```

Reply:
Done